### ISSUE TITLE
VSCode Error: Windows support

### ISSUE THREAD

**Issue by Maxiviper117** [robsontenorio/voltage#1]
Just installed and get error.

This is from vs-code insiders but same issue on vscode 
```bash
[Error - 2:40:30 PM] Voltage client: couldn't create connection to server.
Launching server using command c:\Users\david\.vscode-insiders\extensions\robsontenorio.voltage-0.0.10\voltage failed. Error: spawn c:\Users\xxx\.vscode-insiders\extensions\robsontenorio.voltage-0.0.10\voltage ENOENT
```

Version: 1.95.3 (user setup)
Commit: f1a4fb101478ce6ec82fe9627c43efbf9e98c813
Date: 2024-11-13T14:50:04.152Z
Electron: 32.2.1
ElectronBuildId: 10427718
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by Maxiviper117** [comment]
Voltage extension version 0.0.10

**Comment by robsontenorio** [comment]
@Maxiviper117  I have published 0.0.11 now. Please, give a try.

**Comment by mustaqim224** [comment]
Also have the same error on 0.0.11

[Error - 2:16:57 PM] Voltage client: couldn't create connection to server.
Launching server using command c:\Users\mustaqim\.vscode\extensions\robsontenorio.voltage-0.0.11\voltage failed. Error: spawn

**Comment by robsontenorio** [comment]
Does that log come from this output for Voltage ?

![](https://raw.githubusercontent.com/robsontenorio/voltage/refs/heads/main/output.png)

**Comment by robsontenorio** [comment]
Ok, I realized you are using windows.

**Comment by robsontenorio** [comment]
@Maxiviper117  @mustaqim224  Try now 0.0.12

**Comment by Maxiviper117** [comment]
Still same issue with 0.0.12

![image](https://github.com/user-attachments/assets/a391720e-27c8-4a7f-933b-c9e2fc3713b5)

**Comment by robsontenorio** [comment]
@Maxiviper117  does your workspace have a up and running project? Does it have php installed?

What is output of `php -m` ?

**Comment by ozerozay** [comment]
PHP 8.3.14 (cli) (built: Nov 19 2024 15:53:36) (NTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.3.14, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.14, Copyright (c), by Zend Technologies

**Comment by Maxiviper117** [comment]
@robsontenorio 

Using 

```bash
->php -v

PHP 8.3.14 (cli) (built: Nov 19 2024 15:53:36) (NTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.3.14, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.14, Copyright (c), by Zend Technologies
```
```bash
php -m
[PHP Modules]
bcmath
bz2
calendar
Core
ctype
curl
date
dom
exif
FFI
fileinfo
filter
gd
gmp
hash
iconv
intl
json
libxml
mbstring
mongodb
mysqli
mysqlnd
openssl
pcre
PDO
pdo_mysql
pdo_pgsql
pdo_sqlite
pgsql
Phar
random
readline
redis
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
```

**Comment by robsontenorio** [comment]
@ozerozay @Maxiviper117 @mustaqim224 

This is weird , because I set up a VM with Windows 11 + PHP and works fine. 

What do you guys get when running this on powershell ?

```
php C:\Users\path-to-voltage-binary 
```

This path can be seen on your comments.

**Comment by ozerozay** [comment]
> php voltage

  voltage  0.0.14

  USAGE:  <command> [options] [arguments]

  lsp:start Start Voltage LSP server

> php voltage lsp:start
Content-Length: 88

{"method":"window\/logMessage","params":{"type":3,"message":"\"Voltage starting ...\""}}

**Comment by Maxiviper117** [comment]
![image](https://github.com/user-attachments/assets/321ae04c-0075-4642-8bcf-c2eb97338339)

**Comment by robsontenorio** [comment]
@Maxiviper117  it should be exactly “php voltage”

**Comment by Maxiviper117** [comment]
`php ./voltage` works the same as `php voltage`

![image](https://github.com/user-attachments/assets/5e648c52-b0bb-486e-8ca4-e68291e6e84e)

**Comment by robsontenorio** [comment]
As I am not a Windows expert, I don't know if there is some permission/environment setup I am not aware. 

This plugin tries to run this command in background, through JS.

```
php -d error_reporting=0 /path/to/voltage lsp:start
```

**Comment by Maxiviper117** [comment]
I've figured it out, I believe. I was using Laravel Herd for my sites, which manages its own php.exe files. I had previously removed PHP from the Windows PATH, but Laravel Herd seems to bind its own directory at C:\Users\xxx\.config\herd\bin, which contains all the PHP executables managed by Herd. This allows me to access the PHP command in the terminal without explicitly setting a PHP executable path on the system path. 

Now that I've added the php.exe location to the path, the extension is working.
